src/ui-kit/Pagination/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import { setCurrentPage } from 'redux/actions'
+import { Pagination } from 'antd'
+import './pagination.scss'
```

Comment:
please always separate the styles imports from the others.
Can you see that it is a bit unclear ? it nees an empty line between like that: 
```
import { Pagination } from 'antd'

import './pagination.scss'
```